Support super_read_only Flag as Environmental Variable

**Description**

I have setup a master-slave environment, after finishing it, I tried to execute a write query on the slave and it was executed successfully since the user I was using is the root user with super privileges, and that was making a problem with the replication as the slave will not be able again to replicate with the master, since they will have a conflict in the IDs on the same table. 
After some research I have found that we can block the write query on the slave by updating the super_read_only flag to 1 : 
`SET GLOBAL super_read_only = 1;`

I was wondering if we can pass this config as an environmental variables when starting the slave container, or maybe it would be by default as that when starting a slave container. 

**Steps to reproduce the issue:**

1. Setup Master/Slave Environment 
2. Execute a write query on master and check that it has been replicated to slave.
3. Execute a write query on slave and it will be executed successfully, but not replicated on master (off course) . 
4. Execute write query on master again and check that it will not be replicated on slave for conflicting in the IDs, and the replication will be stopped, you can check the logs on the slave for more info about the error. 

**Describe the results you received:**

The replication between master and slave stopped after following the above steps. 
check the below logs from the slave node: 
```
2020-08-03T06:48:26.920258Z 6 [ERROR] [MY-010584] [Repl] Slave SQL for channel '': Could not execute Write_rows event on table limit_engine.test; Duplicate entry '3' for key 'test.PRIMARY', Error_code: 1062; handler error HA_ERR_FOUND_DUPP_KEY; the event's master log mysql-bin.000025, end_log_pos 437, Error_code: MY-001062
2020-08-03T06:48:26.922325Z 6 [Warning] [MY-010584] [Repl] Slave: Duplicate entry '3' for key 'test.PRIMARY' Error_code: MY-001062
2020-08-03T06:48:26.922353Z 6 [ERROR] [MY-010586] [Repl] Error running query, slave SQL thread aborted. Fix the problem, and restart the slave SQL thread with "SLAVE START". We stopped at log 'mysql-bin.000025' position 156
```

**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           19.03.12
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        48a66213fe
 Built:             Mon Jun 22 15:46:54 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.12
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       48a66213fe
  Built:            Mon Jun 22 15:45:28 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 2
  Running: 2
  Paused: 0
  Stopped: 0
 Images: 4
 Server Version: 19.03.12
 Storage Driver: overlay2
  Backing Filesystem: xfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 3.10.0-1127.13.1.el7.x86_64
 Operating System: CentOS Linux 7 (Core)
 OSType: linux
 Architecture: x86_64
 CPUs: 1
 Total Memory: 487MiB
 Name: centos-slave
 ID: DH3M:2GXT:5S7K:RKGL:INMP:6TKB:XHJG:DWU4:KN4S:V7SI:5OVH:5MOU
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
```

- Output of `docker-compose version` (if applicable):

```
  Not using docker compose
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
I have setup the Env. using virtual box and vagrant.

hi @dani8art 
please check the log below: 
```
mysql 09:55:28.86 Welcome to the Bitnami mysql container
mysql 09:55:28.86 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-mysql
mysql 09:55:28.86 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-mysql/issues
mysql 09:55:28.86 
mysql 09:55:28.86 INFO  ==> ** Starting MySQL setup **
mysql 09:55:28.87 INFO  ==> Validating settings in MYSQL_*/MARIADB_* env vars
mysql 09:55:28.88 INFO  ==> Initializing mysql database
mysql 09:55:28.88 DEBUG ==> Ensuring expected directories/files exist
mysql 09:55:28.90 INFO  ==> Updating 'my.cnf' with custom configuration
mysql 09:55:28.90 INFO  ==> Setting user option
mysql 09:55:28.90 DEBUG ==> Setting user to 'mysql' in mysql configuration file /opt/bitnami/mysql/conf/my.cnf
mysql 09:55:28.91 DEBUG ==> Cleaning data directory to ensure successfully initialization
mysql 09:55:28.91 INFO  ==> Installing database
2020-08-04T09:55:28.967558Z 0 [System] [MY-013169] [Server] /opt/bitnami/mysql/bin/mysqld (mysqld 8.0.21) initializing of server in progress as process 38
2020-08-04T09:55:28.969255Z 0 [Warning] [MY-013242] [Server] --character-set-server: 'utf8' is currently an alias for the character set UTF8MB3, but will be an alias for UTF8MB4 in a future release. Please consider using UTF8MB4 in order to be unambiguous.
2020-08-04T09:55:28.969260Z 0 [Warning] [MY-013244] [Server] --collation-server: 'utf8_general_ci' is a collation of the deprecated character set UTF8MB3. Please consider using UTF8MB4 with an appropriate collation instead.
2020-08-04T09:55:28.976956Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
2020-08-04T09:55:30.062042Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.
2020-08-04T09:55:30.997068Z 6 [Warning] [MY-010453] [Server] root@localhost is created with an empty password ! Please consider switching off the --initialize-insecure option.
mysql 09:55:33.25 INFO  ==> Starting mysql in background
2020-08-04T09:55:33.650227Z 0 [System] [MY-010116] [Server] /opt/bitnami/mysql/bin/mysqld (mysqld 8.0.21) starting as process 94
2020-08-04T09:55:33.675811Z 0 [Warning] [MY-013242] [Server] --character-set-server: 'utf8' is currently an alias for the character set UTF8MB3, but will be an alias for UTF8MB4 in a future release. Please consider using UTF8MB4 in order to be unambiguous.
2020-08-04T09:55:33.676025Z 0 [Warning] [MY-013244] [Server] --collation-server: 'utf8_general_ci' is a collation of the deprecated character set UTF8MB3. Please consider using UTF8MB4 with an appropriate collation instead.
2020-08-04T09:55:33.822235Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
2020-08-04T09:55:35.765310Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.
2020-08-04T09:55:36.640864Z 0 [System] [MY-011323] [Server] X Plugin ready for connections. Bind-address: '::' port: 33060, socket: /tmp/mysqlx.sock
2020-08-04T09:55:38.373848Z 0 [Warning] [MY-010068] [Server] CA certificate ca.pem is self signed.
2020-08-04T09:55:38.374514Z 0 [System] [MY-013602] [Server] Channel mysql_main configured to support TLS. Encrypted connections are now supported for this channel.
2020-08-04T09:55:38.407405Z 0 [ERROR] [MY-010147] [Server] Too many arguments (first extra is '"--super-read-only=ON"').
2020-08-04T09:55:38.407951Z 0 [ERROR] [MY-010119] [Server] Aborting
2020-08-04T09:55:40.268524Z 0 [System] [MY-010910] [Server] /opt/bitnami/mysql/bin/mysqld: Shutdown complete (mysqld 8.0.21)  Source distribution.
```

--- a/5.7/debian-10/rootfs/opt/bitnami/scripts/mysql/entrypoint.sh
+++ b/5.7/debian-10/rootfs/opt/bitnami/scripts/mysql/entrypoint.sh
@@ -16,7 +16,7 @@ set -o pipefail
 
 print_welcome_page
 
-if [[ "$*" = "/opt/bitnami/scripts/mysql/run.sh" ]]; then
+if [[ "$1" = "/opt/bitnami/scripts/mysql/run.sh" ]]; then
     info "** Starting MySQL setup **"
     /opt/bitnami/scripts/mysql/setup.sh
     info "** MySQL setup finished! **"

--- a/8.0/debian-10/rootfs/opt/bitnami/scripts/mysql/entrypoint.sh
+++ b/8.0/debian-10/rootfs/opt/bitnami/scripts/mysql/entrypoint.sh
@@ -16,7 +16,7 @@ set -o pipefail
 
 print_welcome_page
 
-if [[ "$*" = "/opt/bitnami/scripts/mysql/run.sh" ]]; then
+if [[ "$1" = "/opt/bitnami/scripts/mysql/run.sh" ]]; then
     info "** Starting MySQL setup **"
     /opt/bitnami/scripts/mysql/setup.sh
     info "** MySQL setup finished! **"

